### PR TITLE
Metrics Comparison View

### PR DESCRIPTION
**Pull request by Fine0830**
Service, service instance, endpoint and database are the target entities, the user could choose two or more of them. The selector should support choose different types, such as 
(1) one service vs one instance V.S. one endpoint
(2) one service V.S. the other service

### CODE REVIEW

File: vue.config.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ module.exports = {
   devServer: {
     proxy: {
       '/graphql': {
-        target: 'http://127.0.0.1:12800',
+        target: 'http://122.112.182.72:8080/',
```
Comment (x22x22) [inline vue.config.js]: heihei

**Comment by Fine0830** [comment]
HI All,
There is two pictures. We are able to learn more about this PR by pictures.

picture
metric

**Comment by TinyAllen** [comment]
### Bug1 
huge size negative number
image

### Bug2

if I click refresh the metrics select will be change default 

image
image
image

**Comment by Fine0830** [comment]
Bugs are fixed. please check them.

**Pull request merged**